Expo1900/Expo1900/itemDetailViewController.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  itemDetailViewController.swift
+//  Expo1900
+//
+//  Created by 사파리, 파프리 on 2022/04/15.
+//
+
+import UIKit
+
+final class itemDetailViewController: UIViewController {
+    var item: Heritage?
+    
+    @IBOutlet weak var itemImageView: UIImageView!
+    @IBOutlet weak var ItemDescriptionLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpView()
+    }
+    
+    func setUpView() {
+        title = item?.title
+        itemImageView.image = UIImage(named: item?.imageName ?? "swift")
+        ItemDescriptionLabel.text = item?.description
+    }
```

Comment:
세팅되어야 하는 프로퍼티 모두 item이 Nil이 아닐때여야합니다 
따라서 이런 형태로 변경은 어떨까용? (메서드를 호출하는 쪽에서 item의 nil체크를 해서 넣어줄 수 있도로록)
```suggestion
    func setUp(with item: Heritage ) {
        title = item.title
        itemImageView.image = UIImage(named: item.imageName ?? "swift")
        ItemDescriptionLabel.text = item.description
    }
```

& ) 데이터가 Nil일 경우 셋팅되어야하는 디폴트이미지는 placeholder와 같이 따로 선언해놓고 쓰는걸 추천드립니다 ㅎㅎ 이런 string은 왠만하면 직접쓰지않는걸 추천드령요